app/extensions/brave/locales/en-US/menu.properties

```diff
@@ -4,6 +4,7 @@ actualSize=Actual Size
 addBookmark=Add Bookmark…
 addFolder=Add Folder…
 addToFavoritesBar=Add to Favorites Bar
+addToPublisherList=Add to Publisher List
```

Comment:
is it clear that people will know what this is referring to? cc @bradleyrichter @alexwykoff

Reply:
This needs a different string like "Include this site in Brave Payments contributions."

If that is too long for a menu, "Include site in Brave Payments"

Reply:
@bradleyrichter for comparison's sake:

<img width="256" alt="image" src="https://user-images.githubusercontent.com/8732757/39014603-aac98978-43cf-11e8-88bc-8dcd9a1d8990.png">

<img width="256" alt="image" src="https://user-images.githubusercontent.com/8732757/39014605-ae42904a-43cf-11e8-80d7-0397788134e3.png">

Reply:
@bradleyrichter for comparison's sake:

<img width="318" alt="screen shot 2018-04-19 at 12 44 52 pm" src="https://user-images.githubusercontent.com/8732757/39014633-c6ca3046-43cf-11e8-83f2-0ad0e4565781.png">

<img width="225" alt="screen shot 2018-04-19 at 12 46 03 pm" src="https://user-images.githubusercontent.com/8732757/39014637-caa37524-43cf-11e8-9203-d7c31942bded.png">

Reply:
I would go with short version